### PR TITLE
Docker for Song - Configuration1

### PR DESCRIPTION
**Pull request by rtisma**

### CODE REVIEW

File: docker/auth/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM song_base-ubuntu:latest
+MAINTAINER ICGC <dcc-support@icgc.org>
+
+ENV AUTH_INIT $DCC_HOME/init
+ENV AUTH_DATA $DCC_HOME/auth_data
+ENV SIMPLE_AUTH_ROOT $DCC_HOME/SimpleAuth
+
+ADD init $AUTH_INIT
+
+RUN git clone https://github.com/andricDu/SimpleAuth $SIMPLE_AUTH_ROOT && \
```
Comment (andricDu) [inline docker/auth/Dockerfile]: 😊

File: docker/create-buckets-aws/scripts/bucket-setup.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
+
+import boto3
+import os
+
+class S3Client:
+    def __init__(self, host, port, access_key, secret_key, data_bucket, state_bucket, sentinel_key):
```
Comment (andricDu) [inline docker/create-buckets-aws/scripts/bucket-setup.py]: see if you can run pylint or some other linter/checker so you can get a PEP8 check on your python code.

File: docker/create-buckets-aws/scripts/bucket-setup.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    access_key = get_env('MINIO_ACCESS_KEY')
+    secret_key = get_env('MINIO_SECRET_KEY')
+    data_bucket = get_env('STORAGE_SERVER_DATA_BUCKET')
+    state_bucket = get_env('STORAGE_SERVER_STATE_BUCKET')
+    object_sentinel = get_env('STORAGE_SERVER_OBJECT_SENTINEL')
+    data_dir = get_env('STORAGE_SERVER_DATA_DIR')
+
+    sentinel_key = data_dir+'/'+object_sentinel
+
+    s3Client = S3Client( host,
```
Comment (andricDu) [inline docker/create-buckets-aws/scripts/bucket-setup.py]: 👍

**Comment by rtisma** [comment]
ready for merge now. Was going to include documentation, but there is more than i anticipated, so i will make a separate PR for docs

**Pull request merged**